cms/templates/settings.html

```diff
@@ -85,7 +85,7 @@ <h2 class="title-2">${_("Basic Information")}</h2>
             </li>
           </ol>
 
-          % if about_page_editable:
+          % if not marketing_enabled:
```

Comment:
I realized that when I switched from ENABLE_MKTG_SITE to ENABLE_PUBLISHER on this page, I should have left these two messages at the top tied to the marketing site. So switching them back.